index.html

```diff
@@ -0,0 +1,209 @@
+          <li><a href="#">News</a></li>
+          <li><button type="button">Contact Us</button></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="home-intro flex">
+        <div class="intro-heading">
+          <h2>Welcome! Food Lovers!</h2>
+          <h1>
+            Rejovan's Kitchen <br />
+            Serves Delicious <br />
+            Meal
+          </h1>
+        </div>
+        <p class="mobile-paragraph">
+          We give you the best meal you can ever think of, our customer is our
+          priority, come lets give you that special treat you crave for.
+        </p>
+        <h2 class="mobile-date">we open Mon(8:am-8:pm)- Sun(12noon-9:pm)</h2>
+        <p class="mobile-location">
+          located at arkmill road, iyana-ipaja Lagos State.
+        </p>
+      </section>
+
+      <section class="mainprogram flex">
+        <div class="header flex">
+          <h3>our feature</h3>
+          <hr />
+        </div>
+
+        <div class="activities flex">
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/drinks-1.png" alt="featureicon" />
+              <h3>Food</h3>
+            </div>
+            <p>we serve delicious meal that will satisfy your cravings</p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/chef-icon.png" alt="featureicon" />
+              <h3>Chefs</h3>
+            </div>
+            <p>
+              we have well trained chefs that are very professional both cooking
+              and serving.
+            </p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/dish-plate.png" alt="featureicon" />
+              <h3>Customer service</h3>
+            </div>
+            <p>we have a well relateable customer services.</p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/kitchenware_icon.png" alt="featureicon" />
+              <h3>Equipment</h3>
+            </div>
+            <p>
+              our cooking materials are well structured and gotten from the
+              best. And also a neat working environment.
+            </p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/cook-pot.png" alt="featureicon" />
+              <h3>Feedback</h3>
+            </div>
+            <p>we've had a lot of good reports from our costumers.</p>
+          </div>
+        </div>
+
+        <button type="button">See other feedback</button>
+        <a href="#" class="desktop-mainprogramBtn">place your order</a>
+      </section>
+    </main>
+
+    <section class="section-menu flex">
+      <div class="flex menuHeader">
+        <h3>our Menu</h3>
```

Comment:
- [ ] Please, consider adding a background image to the foods, as it was required by the previous reviewer.

Reply:
@elmar8287 please I need clarity on this because in the original project there no background image, except you are talking about the check image if that is what you mean't I feel that is optional. I will await your kind response thanks

Reply:
I see you have fixed it - nice job 👍